client/src/pages/Login.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+/* ---------- CUSTOM COMPONENTS---------- */
+import Container from "../components/Container";
+import Item from "../components/Item";
```

Comment:
These two are 1) very vague names and 2) the components don't seem to be doing much vs just using grid. Do we need to have them as separate components?

Reply:
I made these to take advantage of default props and provide better readability. 

For example, I use the same spacing for Grid container quite often and supply xs={12} for a lot of Grid items so having them as a default prop is useful.

Also, it looks a bit neater to have Container/Item tags.